server/service/devices.go

```diff
@@ -739,7 +792,9 @@ func getDeviceMDMManualEnrollProfileEndpoint(ctx context.Context, request interf
 
 func (svc *Service) GetDeviceMDMAppleEnrollmentProfile(ctx context.Context) (*url.URL, error) {
```

Comment:
Do we need this endpoint exposed to this auth at all? IIRC iOS/iPadOS enrollment has its own completely different flow and that endpoint doesn't follow this path at all...and for iOS/iPadOS you'd need to be enrolled via profile or ABM in order to make it past that auth, which defeats the purpose of the endpoint.

IIRC this endpoint gets used for macOS machines in particular that were nerolled via osquery/fleetd in order to get them added into MDM, so it's not relevant here.